blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -1,6 +1,6 @@
 import keymirror from 'keymirror';
 
-import fetch from 'isomorphic-fetch';
+import fetch from '../util/fetch';
```

Comment:
@scherler that's a good point, it was a long running work in progress, I'll scale back the fetch refactor, it's not really necessary